script/start_new_week.rb

```diff
@@ -0,0 +1,2 @@
+UserAvailability.destroy_all
```

Comment:
The TimeBlocks should have a datetime which have been set to early November. For continuity, each TimeBlock needs to be updated by adding 1 week to their time and saved. Originally,  I wasn't sure whether we would ever require a backlog of blocks, but I put in the infrastructure there, just incase.